src/cdk-experimental/listbox/listbox.spec.ts

```diff
@@ -0,0 +1,102 @@
+import {
+  ComponentFixture,
+  async,
+  TestBed,
+} from '@angular/core/testing';
+import {Component, DebugElement} from '@angular/core';
+import {By} from '@angular/platform-browser';
+import {
+  CdkOption,
+  CdkListboxModule, CdkListbox
+} from './index';
+import {dispatchMouseEvent} from '@angular/cdk/testing/private';
+
+describe('CdkOption', () => {
+
+  describe('selection state change', () => {
+    let fixture: ComponentFixture<ListboxWithCdkOptions>;
+    let listbox: DebugElement;
+    let listboxInstance: CdkListbox;
+    let options: DebugElement[];
+    let optionInstances: CdkOption[];
+    let optionElements: Element[];
```

Comment:
```suggestion
    let optionElements: HTMLElement[];
```
You can typically be a little bit more specific with these and specify `HTMLElement`